### DISCUSSION TITLE
Bankrupt Brand: Keep or Remove?

### DISCUSSION THREAD

**Discussion by jdcarls2** [osmlab/name-suggestion-index#5584] [Q&A] [answered]
Is there particular value in keeping a brand in the NSI that has gone bankrupt? I'm asking specifically in regards to [Bealls (Texas)](https://en.wikipedia.org/wiki/Bealls_(Texas)), which went bankrupt last year and, according to Wikipedia, began liquidating locations.

There might still be Bealls locations out there for all I know, both IRL and in OSM. But is maintaining an entry in the NSI worthwhile? The brand will not be adding any new locations, and any existing locations that don't have the full tagging from the NSI won't be much improved by including the tags (via an iD validator fix, say), if the location is soon to be or already has been liquidated.

I'm thinking it should just be nuked, but I'd like some input before taking a step like that.

**Comment by bhousel** [comment]
I do think it's worth keeping defunct brands in the index for a little while until the map catches up to the business situation.  

If someone does want to remove all liquidated Bealls with a mass edit or manual review each one, the `brand:wikidata` tag makes it easy to find them all.  I guess what I'm saying is there's no harm keeping them in the index for 1-2 years, but I don't think we need to keep them around for 5-10 years, and I dont think we need to keep them in the index past the point that they've been removed from the map..

**Reply by codeinabox** [reply]
How about keeping the brand in the index but emptying the location set?

**Comment by UKChris-osm** [comment, marked as answer]
Could we make it so brands that are defunct can be marked up as such, and have iD highlight a brand match as "_this brand may no longer exist_"?

**Reply by bhousel** [reply]
Yes - it would be a change on the iD side but we do have all the info needed to issue that warning 👍

**Reply by mnalis** [reply]
I see that nowadays https://nsi.guide/ allow filtering by _"Dissolved"_? But I'm not sure how that works, by querying for [P576](https://www.wikidata.org/wiki/Property:P576) _"dissolved, abolished or demolished date"_ on wikidata entry?

**Reply by arrival-spring** [reply]
Yes, see https://github.com/osmlab/name-suggestion-index/blob/8e2c7c179fb1494dc683f216ee20fa632fe60f4d/scripts/build_wikidata.js#L389 and following
